app/Http/Controllers/Sponsors.php

```diff
@@ -2,18 +2,21 @@
 
 namespace App\Http\Controllers;
 
+use \RecursiveIteratorIterator,RecursiveArrayIterator;
 use App\Models\Sponsor;
 use App\Models\SponsorAgent;
 use App\Models\SponsorDetail;
 use App\User;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Storage;
 use App\Http\Resources\Sponsor as SponsorResource;
 use App\Helpers\S3Management;
 use App\Helpers\Auth0Management;
 
 
+
```

Comment:
Are you using an autoformatter, as I don't think this newline is necessary

Reply:
Removed.